plugins/jobs/src/js/components/JobsForm.tsx

```diff
@@ -73,6 +80,10 @@ class JobsForm extends React.Component<JobsFormProps> {
   constructor(props: Readonly<JobsFormProps>) {
     super(props);
 
+    this.state = {
+      editorWidth: undefined
```

Comment:
Oh, interesting... Is this an equivalent TS interface?

```
interface JobsFormState {
  editorWidth?: number;
}

interface JobsFormState {
  editorWidth: number | undefined;
}
```

I would have assumed this is a typescript error because the editorWidth key exists but undefined is not a number.

Reply:
Yea, I thought `key?: number` was the same as `key: number | undefined`. If you think `editorWidth: number | undefined;` is clearer, I'll gladly change it :)